modules/serverless-config-replicas-eventing.adoc

```diff
@@ -45,3 +45,43 @@ spec:
   high-availability:
     replicas: 3
 ----
+
+. You can also specify the number of replicas for a specific workload:
++
+[NOTE]
+====
+Workload-specific configuration overrides the global setting for Knative Eventing.
+====
++
+.Example YAML
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  high-availability:
+    replicas: 3
+  workloads:
+  - name: mt-broker-filter
+    replicas: 3
+----
+
+. Verify that the high availability limits are respected:
++
+.Example command
+[source,terminal]
+----
+$ oc get hpa -n knative-eventing
+----
++
+.Example command
+[source,terminal]
+----
+NAMESPACE                 NAME                     REFERENCE                           TARGETS   MINPODS   MAXPODS   REPLICAS   AGE
+knative-eventing          broker-filter-hpa        Deployment/mt-broker-filter         0%/70%    3         12        2          17m
+knative-eventing          broker-ingress-hpa       Deployment/mt-broker-ingress        0%/70%    2         11        2          17m
```

Comment:
Am I missing something? The minpods==2 and it doesn't match high-availability.replicase==3
It might be confusing.

Reply:
Same for eventing-webhook below.

Reply:
Both should be 3.

Reply:
@mgencur Does this LGTY now?